testing/matrix-sdk-integration-testing/src/helpers.rs

```diff
@@ -14,7 +14,12 @@ use tokio::sync::Mutex;
 static USERS: Lazy<Mutex<HashMap<String, (Client, TempDir)>>> = Lazy::new(Mutex::default);
 
 #[ctor::ctor]
-fn init_logging() {
+fn startup() {
+    init_logging();
+}
+
+/// initialize logging
+pub fn init_logging() {
```

Comment:
Why is the logging initialization logic moved into a public function if that function is never called from outside the module?

Reply:
left over from a refactor that was reverted. well spotted